code/src/LeafletProvider/DrawingTools/DrawMarker.ts

```diff
@@ -0,0 +1,144 @@
+/// <reference path="AbstractProviderTool.ts" />
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+namespace LeafletProvider.DrawingTools {
+    export class DrawMarker extends AbstractProviderTool<Configuration.DrawingTools.DrawMarkerConfig> {
+        private _defaultIcon: L.DivIcon;
+
+        constructor(
+            map: OSFramework.OSMap.IMap,
+            drawingTools: OSFramework.DrawingTools.IDrawingTools,
+            drawingToolsId: string,
+            type: string,
+            configs: Configuration.DrawingTools.DrawMarkerConfig
+        ) {
+            super(
+                map,
+                drawingTools,
+                drawingToolsId,
+                type,
+                new Configuration.DrawingTools.DrawMarkerConfig(configs)
+            );
+
+            this._defaultIcon = new L.DivIcon({
+                iconSize: [24, 40],
```

Comment:
What do you think of creating constant variables for these "magic numbers"?

Reply:
I'll do this.